docs/deployment/kubernetes.md

```diff
@@ -69,7 +69,30 @@ This step assumes you are running `kubectl` on a master host.
 
     !!! note
         For deploying on a cloud that supports Kubernetes *LoadBalancers* you may also want to apply the configuration in: `cloud/lb.yml`.
+        
+        
+    Verify the deployment:
+    
+    ```bash
+    $ kubectl --namespace=openfaas get deploy
```

Comment:
This looks like it would be OK to add, but may already be slightly out of date - faas-netesd is now embedded within the gateway Pod.